src/debug2.cls.php

```diff
@@ -48,6 +48,51 @@ public function __construct() {
 		defined('LSCWP_DEBUG_EXC_STRINGS') || define('LSCWP_DEBUG_EXC_STRINGS', $this->conf(Base::O_DEBUG_EXC_STRINGS));
 	}
 
+	/**
+	 * Disable all functionalities for a time period.
+	 *
+	 * @since 7.4
+	 * @access public
+	 * @param  integer $time How long should we disable LSC functionalities.
+	 */
+	public static function temporary_disable( $time = 86400 ) {
+		$conf = Conf::cls();
+		$disabled = self::cls()->conf( Base::DEBUG_TMP_DISABLE );
+
+		if ( 0 === $disabled ) {
```

Comment:
@timotei-litespeed If it is still in tmp disable, should still allow a refresh to extend to another 24h